aula-pratica-03/ex10.c

```diff
@@ -9,17 +9,13 @@
 int main()
 {
     int vetor[10], num, index_z = 9, index = 0;
-
     for(int i = 0; i < 10; i++) {
         printf("Digite um numero: ");
         scanf("%d", &num);
         vetor[(num == 0) ? index_z-- : index++] = num ;
      }
-
     for(int i  = 0; i < 10; i++) {
         printf("vetor[%d] = %d\n", i, vetor[i]);
     }
-
```

Comment:
eu não to falando pra vc colocar de volta, as ter espaços lógicos assim é bom, só não é legal ter os espaços a mais no final do arquivo